src/utils/OpenSwathWorkflow.cpp

```diff
@@ -499,7 +499,7 @@ class TOPPOpenSwathWorkflow
       feature_finder_param.setValue("TransitionGroupPicker:PeakPickerMRM:method", "corrected");
       feature_finder_param.setValue("TransitionGroupPicker:PeakPickerMRM:signal_to_noise", 0.1);
       feature_finder_param.setValue("TransitionGroupPicker:PeakPickerMRM:gauss_width", 30.0);
-      feature_finder_param.setValue("uis_threshold_sn",-1);
+      feature_finder_param.setValue("uis_threshold_sn",0);
```

Comment:
This updated parameter will reduce the output file size substantially by omitting transitions below the threshold.